AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/utils/Utils.java

```diff
@@ -22,11 +25,11 @@
 import java.util.List;
 
 public class Utils {
-  private Utils() {
-  }
+  private Utils() {}
 
   /**
    * resID = getResId("icon", R.drawable.class);
+   *
    * @param resName
```

Comment:
No need to add @param sections if no detailed comment for them

Reply:
Thanks for the info. I suppose I missed the comments. will add.